app.txt

```diff
@@ -1 +1,5 @@
 this is simple text
+
+
```

Comment:
hey shilpi, why you have add extra space

Reply:
because it was needed here